ui/tests/acceptance/policies-test.js

```diff
@@ -0,0 +1,92 @@
+import { module, test } from 'qunit';
+import { visit, currentURL, click, typeIn, findAll } from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
+import { allScenarios } from '../../mirage/scenarios/default';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import percySnapshot from '@percy/ember';
+import a11yAudit from 'nomad-ui/tests/helpers/a11y-audit';
+
+module('Acceptance | policies', function (hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+
+  test('Policies index route looks good', async function (assert) {
+    assert.expect(4);
+    allScenarios.policiesTestCluster(server);
+    window.localStorage.nomadTokenSecret = server.db.tokens[0].secretId;
```

Comment:
`suggestion`: You're setting the `nomadTokenSecret` in each test but then you're not resetting the value back later. This could leak into other tests and be a hard bug to squash later. Imagine an authentication-related test running after this.